.github/workflows/build.yaml

```diff
@@ -0,0 +1,39 @@
+name: build
+
+on: [push, pull_request]
+jobs:
+  build:
+    name: build
+    strategy:
+      matrix:
+        go-version: [1.16, 1.17, 1.18]
+        os: [ubuntu-latest]
+    runs-on:  ${{ matrix.os }}
+    steps:
+
+      - name: Set up Go 1.17
```

Comment:
name should change with matrix.go-version

Reply:
done